docusaurus/docs/user-docs/content-manager/reviewing-content.md

```diff
@@ -3,16 +3,22 @@ title: Reviewing content
 description: Using review workflows to manage content throughout the content creation process.
 ---
 
-# Reviewing content <EnterpriseBadge /> <AlphaBadge />
+# Reviewing content <EnterpriseBadge /> <BetaBadge />
 
-Using the [review workflows](/user-docs/settings/review-workflows) feature, you can manage content throughout the content creation process. In the Content Manager, you can assign content to a specific stage in the review workflow, and move content between stages as needed.
+Using the [review workflows](/user-docs/settings/review-workflows) feature, you can manage content throughout the content creation process. In the Content Manager, you can change the review stage of the content, and keep moving it between stages as needed.
```

Comment:
Actually, it's in the edit view that we can change the stage.
Also, on the wording, it feels weird to talk about "review stage" even if we talk about "Review Workflow", a stage can be for anything, review, fix, etc.. Shouldn't we keep the "stage" wording?

Reply:
It is indeed in the edit view: I've added this precision in the procedure right below the screenshot. Were you pointing a mistake or were you thinking I should also mention in the introduction that it's in the edit view? I'm not sure 😊 

Also, the "review stage" wording has been discussed with @maevalienard and it's already used in the admin panel so I'm not sure I should change it now.

Reply:
I was thinking of mentioning it also, as it would be confusing if the user stops at the first lines and will say "I don't see where I can change".
For the wording, ok, feels strange but if it has been decided by product design, should be good 👍

Reply:
<img width="345" alt="Capture d’écran 2023-03-28 à 17 35 01" src="https://user-images.githubusercontent.com/54801383/228290750-0fd40e14-e410-45f5-a82b-ae41d1308b4f.png">

Indeed, this wording has been validated during the ideation part :)

Reply:
Alright, I'll add that then! Thanks for the feedback/review 😊